nagios/nrpe.sls

```diff
@@ -2,7 +2,7 @@ include:
   - jcu.repositories.epel
   - jcu.repositories.eresearch
 
-nrpe-plugin:
+nagios-plugins-nrpe:
```

Comment:
This will need a conditional for EL8; the package name must have changed for >= 8

Reply:
I don't know why it isn't showing here but this has been updated to do just that. The state was returned to being called `nrpe-plugin` and a conditional statement around the `-name` tag was added.